src/web/content/static/css/letterpress.css

```diff
@@ -558,16 +556,17 @@ a.sms-large-font:hover {
 }
 
 .sms-feed {
-    width: 8.5rem;
+    width: auto;
```

Comment:
This will make the results text not break into two lines. However, this displays other numbers in, I think, other pages? It will be appreciated if you tell me what other pages I should test where this displays with other values. Probably some admin pages I'm forgetting to test.

Reply:
The results.html template is used on the main search results, and also Browse and Newest. On the main search results and also Newest it shows the total number of results, while on Browse it shows the total number of sites. Since "sites" is shorter than "results" this fix seems to cover both.